watch-history-exporter-for-amazon-prime-video.js

```diff
@@ -35,16 +54,17 @@
 
 	// Add a movie or episode to the array
 	const addItem = (watchHistoryArray, dateWatched, title, episodeTitle) => {
-		const formattedDateWatched = new Date(dateWatched)
-			.toISOString()
+		// const formattedDateWatched = new Date(dateWatched)
```

Comment:
This only works for German now, so we need a way of supporting multiple languages. I would make a `parseDateString(dateString)` function that gets the page locale (ex. `en-us`, `de-de`) with `document.documentElement.lang` and calls the appropriate parsing function for that language with `dateString` to get the Date object.